scripts/amd/docker_run.sh

```diff
@@ -12,11 +12,9 @@ VOLUMES="-v $HOME/dockerx:/dockerx -v /data:/data"
 # WORK_DIR='/root/$(basename $(pwd))'
 WORK_DIR="/dockerx/$(basename $(pwd))"
 
-# IMAGE_NAME=nvcr.io/nvidia/pytorch:21.08-py3
-# IMAGE_NAME=rocm/pytorch:latest 
-IMAGE_NAME=rocm/pytorch:rocm5.4_ubuntu20.04_py3.8_pytorch_1.12.1
-# IMAGE_NAME=rocm/pytorch:rocm4.3.1_ubuntu18.04_py3.6_pytorch_1.10.0
-# IMAGE_NAME=triton_rocm_20-52 # build this docker before running
+IMAGE_NAME=rocm/pytorch-nightly:latest
```

Comment:
For these runs...Should we try to pin a pytorch level so you don't get false positives if something else happens with Pytorch unrelated to this particular problem?

Reply:
this is a helper script. It is not used by the ci. The CI uses pulls the torch wheel. See [here](https://github.com/ROCmSoftwarePlatform/triton/blob/aabc2511f6553c2f26eb9c2aec7cf25f6421baa6/.github/workflows/amd-offline-tests.yml#L52)